src/scss/components/_back-to-top.scss

```diff
@@ -0,0 +1,24 @@
+@use 'abstracts' as *;
+
+.back-to-top {
+  display: none;
+
+  @include breakpoint('large') {
+    position: absolute;
+    right: 40px;
+    display: flex;
+    margin-bottom: rem(40px);
```

Comment:
When positioning fixed or absolute elements, it's best practice to use the top, right, bottom left properties. So I would change this to `bottom: rem(40px)`